Error reported when executing command 'make'

I want to install memtier_benchmark. Command like that:
```
yum install -y autoconf automake make gcc-c++ git
yum install -y pcre-devel zlib-devel libmemcached-devel openssl-devel libevent-devel
export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:${PKG_CONFIG_PATH}
git clone https://github.com/RedisLabs/memtier_benchmark.git
cd memtier_benchmark/
autoreconf -ivf
./configure
make
make install
```
But, when i execute command 'make', i get the error: 
```
[root@controller-node-1 memtier_benchmark]# make
make  all-am
make[1]: Entering directory `/root/memtier_benchmark'
  CXX      memtier_benchmark-memtier_benchmark.o
memtier_benchmark.cpp:887:21: warning: identifier ‘nullptr’ is a keyword in C++11 [-Wc++0x-compat]
                     while (tls_token != nullptr) {
                     ^
memtier_benchmark.cpp: In function ‘int config_parse_args(int, char**, benchmark_config*)’:
memtier_benchmark.cpp:887:41: error: ‘nullptr’ was not declared in this scope
                     while (tls_token != nullptr) {
                                         ^
In file included from memtier_benchmark.cpp:39:0:
memtier_benchmark.cpp: In function ‘int main(int, char**)’:
memtier_benchmark.cpp:1367:50: error: ‘SSL_OP_NO_TLSv1_3’ was not declared in this scope
             SSL_CTX_set_options(cfg.openssl_ctx, SSL_OP_NO_TLSv1_3);
                                                  ^
make[1]: *** [memtier_benchmark-memtier_benchmark.o] Error 1
make[1]: Leaving directory `/root/memtier_benchmark'
make: *** [all] Error 2
```

After modifying the code and excuting the command `make`,  only the following error exists:
```
[root@controller-node-1 memtier_benchmark]# make
make  all-am
make[1]: Entering directory `/root/memtier_benchmark'
  CXX      memtier_benchmark-memtier_benchmark.o
In file included from memtier_benchmark.cpp:39:0:
memtier_benchmark.cpp: In function ‘int main(int, char**)’:
memtier_benchmark.cpp:1367:50: error: ‘SSL_OP_NO_TLSv1_3’ was not declared in this scope
             SSL_CTX_set_options(cfg.openssl_ctx, SSL_OP_NO_TLSv1_3);
                                                  ^
make[1]: *** [memtier_benchmark-memtier_benchmark.o] Error 1
make[1]: Leaving directory `/root/memtier_benchmark'
make: *** [all] Error 2
```

--- a/memtier_benchmark.cpp
+++ b/memtier_benchmark.cpp
@@ -884,7 +884,7 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
                 {
                     const char tls_delimiter = ',';
                     char* tls_token = strtok(optarg, &tls_delimiter);
-                    while (tls_token != nullptr) {
+                    while (tls_token != 0) {
                         if (!strcasecmp(tls_token, "tlsv1"))
                             cfg->tls_protocols |= REDIS_TLS_PROTO_TLSv1;
                         else if (!strcasecmp(tls_token, "tlsv1.1"))
@@ -904,7 +904,7 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
                             return -1;
                             break;
                         }
-                        tls_token = strtok(nullptr, &tls_delimiter);
+                        tls_token = strtok(0, &tls_delimiter);
                     }
                     break;
                 }

